.travis.yml

```diff
@@ -32,6 +33,7 @@ script:
   - if [[ $coverage = 1 ]]; then phpdbg -qrr vendor/bin/behat --profile coverage; else vendor/bin/behat; fi
   - if [[ $coverage = 1 ]]; then phpdbg -qrr phpcov.phar merge --clover build/logs/clover.xml build/cov; fi
   - tests/Fixtures/app/console api:swagger:export > swagger.json && swagger validate swagger.json && rm swagger.json
+  - if [[ $lint = 1 ]]; then php php-cs-fixer.phar fix --dry-run --diff; fi
```

Comment:
> What about not making phpunit/behat run if php-cs-fixer fails?

That would require using `&&`, or writing a shell script: https://docs.travis-ci.com/user/customizing-the-build/#Implementing-Complex-Build-Steps